extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -142,6 +145,32 @@ describes.realWin('consent-ui', {
     });
   });
 
+  describe('placeholder', () => {
+    it('should add a placeholder element' +
```

Comment:
Having placeholder here feels a bit redundant. The test will read as "consent-ui placeholder should add a placeholder element while loading CMP Iframe". Consider something like "consent-ui placeholder should be present while loading CMP Iframe".